src/lib/y2packager/dialogs/addon_selector.rb

```diff
@@ -206,16 +206,17 @@ def continue_msg
       # description widget
       # @return [Yast::Term] the addon details widget
       def details_widget
-        VWeight(40, RichText(Id(:details), Opt(:disabled), "<small>" +
-        description + "</small>"))
+        VWeight(
+          40,
+          RichText(Id(:details), Opt(:disabled), initial_description)
+        )
       end
 
       # extra help text
-      # @return [String] translated text
-      def description
-        # TRANSLATORS: inline help text displayed below the product selection widget
-        _("Select a product to see its description here. The dependent products " \
-          "are selected automatically.")
+      # @return [String] first product description
+      def initial_description
+        return "" if products.empty?
+        product_description(products.first)
```

Comment:
NP: Although the code will work as it is, maybe we could do that in a more explicit way, asking to the MultiSelectionBox by the :CurrentItem and looking for the product to display its description.

Anyway, I have no strong opinion about it and this changes look good enough to me to solve the issue mentioned in the PR description.

Reply:
The problem is that, at this point, the products list is not displayed yet so you will get a `nil` value from `QueryWidget`.